packages/blocks/src/api/constants.js

```diff
@@ -114,6 +114,14 @@ export const __EXPERIMENTAL_STYLE_PROPERTY = {
 		value: [ 'elements', 'link', 'color', 'text' ],
 		support: [ 'color', 'link' ],
 	},
+	buttonColor: {
+		value: [ 'elements', 'button', 'color', 'text' ],
+		support: [ 'color', 'button' ],
+	},
+	buttonBackground: {
+		value: [ 'elements', 'button', 'color', 'background' ],
```

Comment:
I found this confusing. The best way wold be to follow CSS and use `[ 'elements', 'button', 'background', 'color' ]` but that is outside the scope.

Reply:
This is the way that the data is structured in Global styles - properties are grouped by `color`, `typography`, `spacing` etc. It doesn't map onto CSS.